lib/nada_certs/DigiCertCA.crt

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
Its a bad idea to put files outside `lib/vindata` when you are on a gem, because you can collide with the app or other gems, use `lib/vindata/nada_certs/DigiCertCA.crt` instead.

Reply:
Right, wanted to put it into services dir, but pushed the wrong branch somehow. Fixing.